dev/CommonStyles/DatePicker_themeresources.xaml

```diff
@@ -177,7 +172,8 @@
                     <Grid x:Name="LayoutRoot" Margin="{TemplateBinding Padding}">
                         <Grid.Resources>
                             <Style x:Key="DatePickerFlyoutButtonStyle" TargetType="Button">
-                                <Setter Property="UseSystemFocusVisuals" Value="False" />
+                                <Setter Property="UseSystemFocusVisuals" Value="True"/>
```

Comment:
I wonder if we should use the contract test:

<contract6Present:Setter Property="UseSystemFocusVisuals" Value="True" />

Reply:
Updated